scripts/extensions/sams/src/components/assets/assetListPanel.tsx

```diff
@@ -44,10 +46,21 @@ export class AssetListPanel extends React.PureComponent<IProps> {
         this.props.updateSelectedAssetIds(asset);
     }
 
+    onForceUnlock(asset: IAssetItem) {
+        this.props.forceUnlockAsset!(asset);
+    }
+
     render() {
         const {gettext} = superdeskApi.localization;
         const {assertNever} = superdeskApi.helpers;
 
+        if (superdeskApi.privileges.hasPrivilege('sams_manage_assets')) {
+            this.props.actions!.push({
```

Comment:
React properties should not be modified directly.

The `SamsWorkspaceComponent` already constructs a list of actions to provide, it might be better to manage these list of actions there instead.